mlflow/tracking/_tracking_service/client.py

```diff
@@ -106,12 +106,21 @@ def list_run_infos(
             experiment_id, run_view_type, max_results, order_by, page_token
         )
 
-    def list_experiments(self, view_type=None):
+    def list_experiments(
+        self, view_type=None, max_results=SEARCH_MAX_RESULTS_DEFAULT, page_token=None
```

Comment:
BTW note that the value of `SEARCH_MAX_RESULTS_DEFAULT` here is 1000 - I'd like to do a bit of data science to understand existing client usage before settling on this value, just to make sure I understand what impact this might have on existing users (of course from a server-implementor perspective a lower default is better from a load perspective)

Reply:
I'm certainly not set on anything in terms of these limits, I just defaulted to whatever I could find elsewhere. Whenever you have time to poke around and settle on something, let me know!